packages/EasyDecision/tests/Stubs/DecisionStub.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyDecision\Tests\Stubs;
+
+use EonX\EasyDecision\Decisions\AbstractDecision;
+
+final class DecisionStub extends AbstractDecision
+{
+    /**
+     * @var mixed
+     */
+    private $defaultOutput;
+
+    /**
+     * @var mixed
+     */
+    private $output;
+
+    /**
+     * @var null|\Closure
+     */
+    private $outputClosure;
+
+    public function __construct(
```

Comment:
I can't see anywhere in the test where we're using the other constructor arguments, why having them?